.github/workflows/test.yaml

```
@@ -19,18 +19,21 @@ jobs:
 
   environments:
     runs-on: buildjet-4vcpu-ubuntu-2204
-    name: ${{ matrix.env }}
+    name: ${{ matrix.env }} - v${{ matrix.version }}
     strategy:
       fail-fast: false
       matrix:
         env: [node]
+        version: [18.18.2, 20]
```

@danielbate I think we should only consider `18.18.2`.

@nedsalk I think you can add the `20` variation as part of your PR since that is the one that adds `v20` support.

```suggestion
        version: [18.18.2]
```